orm/src/main/java/org/hibernate/search/jpa/FullTextEntityManager.java

```diff
@@ -42,10 +42,13 @@
 
 	/**
 	 * Force the (re)indexing of a given <b>managed</b> object.
-	 * Indexing is batched per transaction: if a transaction is active, the operation
+	 * Indexation is batched per transaction: if a transaction is active, the operation
```

Comment:
why are you changing this? `Indexation` is not the correct term. It needs to be `Indexing`. I now it's wrong in some other places, but don't make it worse :)